### PR TITLE
(#40) Attempt to start CCM Website more gracefully

### LINKED ISSUE #40: Error bringing up CCM Web in SSL script
When running `Set-SSLSecurity.ps1`, starting the `ChocolateyCentralManagement` has failed on one occasion with the following error: 
```powershell
start-webitem : The object identifier does not represent a valid object. (Exception from HRESULT: 0x800710D8)
At C:\choco-setup\files\Set-SSLSecurity.ps1:110 char:5
+     Start-Website ChocolateyCentralManagement
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (:) [Start-WebItem], COMException
+ FullyQualifiedErrorId : InvalidOperation,Microsoft.IIs.PowerShell.Provider.StartItemCommand
```
We may have to build a wait or check into this.

### PR DESCRIPTION
**Pull request by steviecoaster**
There may be a timing issue when bringing up the CCM website, so add a retry
and finally a warning if it fails to come up

Fixes #40

### CODE REVIEW

File: Set-SSLSecurity.ps1 [hunk 1]
```diff
@@ -107,7 +107,17 @@ process {
...
 
     #Start the components back up
-    Start-Website ChocolateyCentralManagement
+    try {
+        Start-Website ChocolateyCentralManagement
+    }
+    catch {
+        #Try again...harder.
+        Write-Warning "Unable to start Chocolatey Central Management website, retrying"
+        Get-Website ChocolateyCentralManagement | Start-Website -ErrorAction Stop
```
Comment (adilio) [inline Set-SSLSecurity.ps1:116]:
```suggestion
        Get-Website ChocolateyCentralManagement | Start-Website
        Start-Sleep -Seconds 5
        Get-Website ChocolateyCentralManagement | Start-Website -ErrorAction Stop
```

I don't know if it's worth adding a third try, with a wait/sleep?

**Review by adilio — CHANGES_REQUESTED** [review]

File: Set-SSLSecurity.ps1 (hunk 1 above)
Comment (steviecoaster) [reply Set-SSLSecurity.ps1:116]: Not sure either. My gut says trying twice will be good enough. However, We could try/catch the same code, then do it by pipeline in the finally, with a warning if that also fails.

Comment (adilio) [reply Set-SSLSecurity.ps1:116]: I'm good with that!

If you put up a commit with those changes, I'll go ahead and approve.

Please and thank you! 🙏🏼 ❤️

Comment (steviecoaster) [reply Set-SSLSecurity.ps1:116]: Try/Catch/Finally isn't gonna work here. I forgot. Finally will fire even after a successful Try {} block. Jig it slightly differently, I shall.

**Comment by adilio** [comment]
LGTM! 🚀

**Review by adilio — APPROVED** [review]

**Review by ryanrichter94 — APPROVED** [review]
LGTM 🚀 Appeciate the good code comments.

**Pull request merged**